src/Commands/Backup/ListCommand.php

```diff
@@ -31,32 +26,21 @@ class ListCommand extends TerminusCommand implements SiteAwareInterface
      * @return RowsOfFields
      *
      * @param string $site_env Site & environment in the format `site-name.env`
-     * @param string $element [code|files|database|db] Backup element filter. If not defined, all elements are selected.
+     * @option string $element [code|files|database|db] Backup element filter
      *
      * @usage <site>.<env> Lists all backups made of <site>'s <env> environment.
      * @usage <site>.<env> --element=<element> Lists all <element> backups made of <site>'s <env> environment.
      */
-    public function listBackups($site_env, $element = 'all')
+    public function listBackups($site_env, array $options = ['element' => 'all',])
```

Comment:
This is something we missed: all the other `backup` commands use element as an option (and rightfully so because it's optional, even here) but `backup:list` has it as a parameter. I would like to change it to an option.